pkg-manager/core/src/install/index.ts

```diff
@@ -141,12 +142,18 @@ type Opts = Omit<InstallOptions, 'allProjects'> & {
   binsDir?: string
 } & InstallMutationOptions
 
+export interface InstallResult {
+  newDefaultCatalogs: Record<string, ResolvedCatalogEntry> | undefined
```

Comment:
This is too limited. Even if we don't support them yet, I would return just "newCatalogs" to support named ones as well.

Reply:
Agree with this!